xml/MAIN-SBP-CSP-UpdateInfra.xml

```diff
@@ -879,24 +900,57 @@ sudo chown -R _rmt:nginx /var/lib/rmt/public/repo
 						<filename>rmt-server-config</filename>.</para>
 				</step>
 				<step>
-					<para> Add an Instance Verification example in
-							<filename>/usr/share/rmt/engines/instance_verification/lib/instance_verification/providers/example_csp.rb</filename>
-						by pasting the following information: </para>
+                                  <!-- TODO: Verify that CSPs are allowed to support BYOS; if not this section is superfluous and should
+                                  be removed. -->
+                                  <para>In cases where Pay As You (PAYG), also referred to as On-Demand, and a
+                                  Bring Your Own Subscription (BYOS) models for SUSE products are offered repository access controll
+                                  must be implemented. Only PAYG instance may access the update infrastructure. Customers using BYOS are
+                                  responsible for registering their instances with SCC, their own RMT or SUSE Manager Servers.</para>
+
+					<para> Instance verification is implemented by placing the implementation into
+					<filename>/usr/share/rmt/engines/instance_verification/lib/instance_verification/providers/</filename>.
+                                        The implementation is in Ruby and needs to verify, based on instance information sent by the
+                                        client, whether or not the client is authorized to access the repositories. The implementation must
+                                        support the <emphasis role="strong">isntance_valid</emphasis> method. The following example provides
+                                        guidance for the implementation of the verification plugin.
+					</para>
+                                        <!-- TODO @jesus to verify the example and push it back to RMT example_verification.rb -->
+                                        <!-- RJS -->
 					<screen>
-class InstanceVerification::Providers::Example &lt; InstanceVerification::ProviderBase
-	def instance_valid?
-		@instance_id = 'i-0000000000'
-		@instance_billing_info = 'example'
-		true # this is an example, always returns true
-	end
-end
+                                          class InstanceVerification::Providers::Example &lt; InstanceVerification::ProviderBase
```

Comment:
@jesusbv Can you please review the example code starting here and if this looks good update the provided example.rb script accordingly? Thanks

Reply:
Done